README.md

```diff
@@ -18,39 +22,16 @@ visiting that repo!
 
 ## 🔮 prerequisities
 
-this project is written in rust. [get rust] to work on this project.
-
-[get rust]: https://www.rustup.rs/
-
-if you want to publish packages, you'll also need an account on [npm] and have
-[node/npm] installed.
-
-[npm]: https://www.npmjs.com
-[node/npm]: https://nodejs.org/
+- [development environment](docs/prerequisites.md)
+- [installation and getting started](docs/setup.md)
 
-## 🏃‍♀️ up and running
+## 🎙️ commands
 
-1. fork and clone this repository
-2. install [node/npm]
-2. `cd wasm-pack`
-3. `cargo run`
+- [`init`](docs/init.md): generate an npm wasm pkg from a rustwasm crate
+- [`pack`](docs/pack.md): create a tarball of your rustwasm pkg
+- [`publish`](docs/publish.md): publish your rustwasm pkg to a registry
 
-## 💃 commands
-- `help`: display available commands
-- 🐣  `init`: create necessary files for js interop and npm publishing
-  - optionally pass a path to a dir that contains a `Cargo.toml`, e.g.:
-    ```
-    wasm-pack init examples/js-hello-world
-    ```
-  - optionally pass a scope name to generate a `package.json` for a scoped pkg, e.g.:
-    ```
-    wasm-pack init examples/scopes-hello-world --scope test
-    ```
-    generates a `package.json` for an npm package called `@test/scopes-hello-world`
-- 🍱  `pack`: create a tarball but don't push to the npm registry (see https://docs.npmjs.com/cli/pack)
-- 🎆  `publish`: create a tarball and publish to the npm registry (see https://docs.npmjs.com/cli/publish)
-
-### logging
+### 📝 logging
```

Comment:
Might as well just break this out into it's own separate file now that it's already done for the commands